tests/e2e/flowtest/mcg/test_mcg_namespace_disruptions_crd.py

```diff
@@ -8,6 +8,7 @@
     skipif_aws_creds_are_missing,
     flowtests,
     skipif_openshift_dedicated,
```

Comment:
I think it would be better to have something like skipif_managed_service for skipping both rosa and osd, as they go together in so many places.